### DISCUSSION TITLE
Is there a working demonstration of DeepAR model on PyTorch

### DISCUSSION THREAD

**Discussion by KexinFeng** [awslabs/gluonts#2420] [Q&A]
Hi, I'm trying to integrate gluonTS with DJL. Is there a working demonstration of DeepAR model on PyTorch? Currently I only find this https://github.com/awslabs/gluonts/blob/dev/examples/m5_gluonts_template.ipynb
But it only works for MxNet DeepAR model.

Thanks

**Comment by jaheba** [comment]
Hi, can you explain what you mean by integrating with DJL? Generally, the PyTorch version works mostly like the MXNet version,
